client/src/sass/overrides.sass

```diff
@@ -102,19 +102,58 @@
 .v-tab:not(.active-tab):hover
   background-color: inherit !important
 
-.v-tooltip, .v-tooltip--bottom
-  background-color: white !important
-  border-radius: 4px
-  border: 1px solid $dk-text !important
+.top-tooltip, .left-tooltip
+  margin-left: 0 !important
 
 .v-tooltip__content
-  background-color: white !important
-  opacity: 1 !important
-  color: $text !important
+  width: 270px !important
+  background-color: RGBA(73,80,87,0.95) !important
+  color: #ffffff !important
   border-radius: 4px
-  box-shadow: 0 0 7px -2px $grey-5
   font-size: 12px !important
+  line-height: 18px !important
   padding: 20px !important
+  letter-spacing: 0
+
+.v-tooltip__content:after
```

Comment:
Why can't v-tooltip props be used to place the tooltip at right/left/top/bottom, possibly with offsets or nudges?

https://vuetifyjs.com/en/api/v-tooltip/

Reply:
I do use those for general alignment, though the positioning isn't quite what the design calls for. This also applies the styling for the pointers.